ingestion/src/metadata/ingestion/source/mssql_usage.py

```diff
@@ -101,6 +106,14 @@ def next_record(self) -> Iterable[TableQuery]:
                 self.report.scanned(f"{row['database_name']}")
             yield table_query
 
+            query_info = {
+                "sql": table_query.sql,
+                "from_type": "table",
+                "to_type": "table",
+                "service_name": self.config.service_name,
+            }
+            ingest_lineage(query_info, self.metadata_config)
+
```

Comment:
It repeats in all the usage connectors, let's create a single method in utils and call that for all the connectors